docs/source/logic.rst

```diff
@@ -128,9 +128,10 @@ applications on the MOI network.
     // Example
     const initWallet = async () => {
         const mnemonic = "mother clarify push liquid ordinary social track ...";
-        const provider = new JsonRpcProvider("http://localhost:1600/");
-        const wallet = new Wallet(provider);
-        await wallet.fromMnemonic(mnemonic);
+        const wallet = await Wallet.fromMnemonic(mnemonic);
+
+        wallet.connect(new JsonRpcProvider("http://localhost:1600/"));
+
```

Comment:
nitpick, please create a variable for provider and use it to enhance the readability.

Reply:
a36b11b60f5391536a65926f9450046806d84589